docker/tds/tds-docker.env

```diff
@@ -5,7 +5,7 @@ MYSQL_ROOT_PASSWORD=
 MYSQL_DATABASE=
 
 # Exam Service DB information
-EXAM_DB_HOST=
+EXAM_DB_HOST=exam_mysql
```

Comment:
Once the related PR linked in the overview is completed, you'll be able to replace this with your machine's intranet IP to connect to a local mysql database.